src/ripple/app/misc/Validations.cpp

```diff
@@ -387,6 +387,31 @@ class ValidationsImp : public Validations
         return ret;
     }
 
+    std::uint32_t getValidationTime (uint256 const& hash, int minValidations)
+    {
+        if (minValidations < 1)
+            minValidations = 1;
+
+        std::vector <std::uint32_t> times;
+
```

Comment:
Does it make sense to call `times.reserve (minValidations);` to reduce the number of calls to the freestore under the lock?